### PR TITLE
angular to react configuration migration guide

### LINKED ISSUE #79200: Angular migration guide: Using the Panel MigrationHandler when upgrading a panel

### PR DESCRIPTION
**Pull request by briangann**
**What this PR does / why we need it**:

Adds how-to guide for angular plugins migrating to react

Fixes grafana/grafana#79200

### CODE REVIEW

**Comment by josmperez** [comment]
Hi, great job. I have completed a first pass editorial review. Please review the latest commit with proposed changes and ensure that no inaccuracies have been introduced. I will begin a second pass later this week or when the PR gets out of "Draft" state.

**Comment by briangann** [comment]
@josmperez ready for a second pass!

**Review by josmperez — CHANGES_REQUESTED** [review]
Hi Brian, I have made a second editorial pass, this one focused on structure (there were some issues with headings not being explanatory enough).

File: docusaurus/docs/migration-guides/angular-react/configuration-migration.md [hunk 1]
```diff
@@ -0,0 +1,214 @@
...
+
+Your plugin can access the variable `config.buildInfo.version` to determine the running version of Grafana. The migration handler can use this value to set valid defaults.
+
+It is possible that multiple versions of Grafana have had a backported patch, and as a result they may have removed a feature that your plugin expected. In the case of the previous example, it is the Roboto font.
+
+The migration handler gets the runtime version and uses semver to determine which font to use. Older versions do not have the newer Inter as a font, and so Roboto is safest to load. Newer releases have removed Roboto, and so the plugin should load Inter instead.
+
+There are two cases here:
+
+#### Case 1: User runs panel with Roboto selected
```
Comment (josmperez) [inline docusaurus/docs/migration-guides/angular-react/configuration-migration.md:82]:
```suggestion
#### Case 2: User runs panel with Roboto selected
```

**Comment by briangann** [comment]
not sure why this is blocked with a request pending, if we're good to go Joe i'll force the merge.

**Comment by josmperez** [comment]
Okay, sounds good

**Pull request merged**